<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QDialog" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background:#ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_19">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="doctor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff;
   color: #fa7014;
  border-style:none none solid none;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-color:#607e8d;
}

QLineEdit::hover {
	border-color:#435863;
	border-width: 2.5px;
}

QLineEdit::pressed{
	border-color:#263238;
  color: #414747; 

}




</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="cabinet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff;
   color: #fa7014;
  border-style:none none solid none;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-color:#607e8d;
}

QLineEdit::hover {
	border-color:#435863;
	border-width: 2.5px;
}

QLineEdit::pressed{
	border-color:#263238;
  color: #414747; 

}




</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="time">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff;
   color: #fa7014;
  border-style:none none solid none;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-color:#607e8d;
}

QLineEdit::hover {
	border-color:#435863;
	border-width: 2.5px;
}

QLineEdit::pressed{
	border-color:#263238;
  color: #414747; 

}




</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="styleSheet">
         <string notr="true">QDateTimeEdit {
  background-color: #ffffff;
   color: #fa7014;
  border-style:none none solid none;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-color:#607e8d;
}

</string>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel {
  color: #414747; 
  border-style:nonet;
  font-family:Verdana;
    text-align:  center;
font-size:10;
font-weight:bold;

}</string>
        </property>
        <property name="text">
         <string>Дата записи:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="patient">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff;
   color: #fa7014;
  border-style:none none solid none;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-color:#607e8d;
}

QLineEdit::hover {
	border-color:#435863;
	border-width: 2.5px;
}

QLineEdit::pressed{
	border-color:#263238;
  color: #414747; 

}




</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_20">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QLineEdit" name="passport">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff;
   color: #fa7014;
  border-style:none none solid none;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-color:#607e8d;
}

QLineEdit::hover {
	border-color:#435863;
	border-width: 2.5px;
}

QLineEdit::pressed{
	border-color:#263238;
  color: #414747; 

}




</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="strahovk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff;
   color: #fa7014;
  border-style:none none solid none;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-color:#607e8d;
}

QLineEdit::hover {
	border-color:#435863;
	border-width: 2.5px;
}

QLineEdit::pressed{
	border-color:#263238;
  color: #414747; 

}




</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="address">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff;
   color: #fa7014;
  border-style:none none solid none;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-color:#607e8d;
}

QLineEdit::hover {
	border-color:#435863;
	border-width: 2.5px;
}

QLineEdit::pressed{
	border-color:#263238;
  color: #414747; 

}




</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel {
  color: #414747; 
  border-style:nonet;
  font-family:Verdana;
    text-align:  center;
font-size:10;
font-weight:bold;

}</string>
        </property>
        <property name="text">
         <string>Дата рождения:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEdit_2">
        <property name="styleSheet">
         <string notr="true">QDateTimeEdit {
  background-color: #ffffff;
   color: #fa7014;
  border-style:none none solid none;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-color:#607e8d;
}

</string>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Добавить нового пациента</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Записать уже существующего</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: #fa7014;
  color: white; 
 border-style: solid;
  border-color: #fa7014;
	border-width:4px;
	border-radius:4px;
}

QPushButton:hover {
    background-color: #fb9b5a;
border-color: #fb9b5a;
}
</string>
        </property>
        <property name="text">
         <string>ОК</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: #fa7014;
  color: white; 
 border-style: solid;
  border-color: #fa7014;
	border-width:4px;
	border-radius:4px;
}

QPushButton:hover {
    background-color: #fb9b5a;
border-color: #fb9b5a;
}
</string>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>doctor</tabstop>
  <tabstop>cabinet</tabstop>
  <tabstop>time</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>patient</tabstop>
  <tabstop>address</tabstop>
  <tabstop>strahovk</tabstop>
  <tabstop>passport</tabstop>
  <tabstop>dateEdit_2</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
